docs/cep/query-guide/table-collection.md

```diff
@@ -325,4 +333,4 @@ CREATE STREAM TempStream (deviceID long, roomNo int, temp double);
 
 insert into ServerRoomTempStream
```

Comment:
@diana-macrometa - Let's use cap letters - `INSERT INTO` and `FROM`/`IN` (below)